src/Collection/Collection.router.ts

```diff
@@ -105,7 +105,7 @@ export class CollectionRouter extends Router {
       typeof is_published === 'undefined'
         ? Collection.find<CollectionAttributes>()
         : Collection.find<CollectionAttributes>({ is_published }),
-      collectionAPI.fetchCollections(),
```

Comment:
I think there's been a mixup with prettier versions as, when I re-save this file I get the changes:
![image](https://user-images.githubusercontent.com/692077/123872642-c224ee80-d90b-11eb-8721-44a9cb568afd.png)

Reply:
mm could be related to this? https://github.com/decentraland/builder-server/blob/master/tslint.json#L4

Reply:
im getting these when i `npm ls prettier`:
```
builder-server@2.10.0 /Users/juancazala/code/builder-server
├─┬ dcl-catalyst-client@4.0.5
│ └─┬ dcl-catalyst-commons@3.0.1
│   └── prettier@2.2.1 deduped
└── prettier@2.2.1
```